ast/format_test.go

```diff
@@ -10,12 +10,9 @@ import (
 )
 
 var skip = map[string]string{
-	"array_expr":            "without pars -> bad syntax, with pars formatting removes them",
-	"conditional":           "how is a conditional expression defined in spec?",
-	"multi_var_decl":        "how is a variable declaration with multiple declarations represented?",
-	"logic":                 "the AST created for logic ops is wrong (https://github.com/influxdata/flux/issues/1077)",
-	"logic with comparison": "the AST created for logic ops is wrong (https://github.com/influxdata/flux/issues/1077)",
-	"logic and math":        "the AST created for logic ops is wrong (https://github.com/influxdata/flux/issues/1077)",
+	"array_expr":     "without pars -> bad syntax, with pars formatting removes them",
+	"conditional":    "how is a conditional expression defined in spec?",
+	"multi_var_decl": "how is a variable declaration with multiple declarations represented?",
```

Comment:
Flux doesn't have a `var` keyword. Can you remove this test?

Reply:
absolutely, we had it in the past 😅